docs/components/concepts/backups.md

```diff
@@ -12,29 +12,47 @@ A Camunda 8 SaaS backup consists of a data backup of Zeebe, Operate, Tasklist, O
 
 With backups, you can capture snapshots of your data and applications while they are actively in use, resulting in zero downtime or disruption to your operations. Backups are designed specifically for disaster recovery purposes, and should not be used for archival of process data.
 
-## Backup and cluster relationship
+## Cluster relationship
```

Comment:
Let's remove this section heading and place all its content into the Caution box. This is a single point and I don't think we need a section here